packages/common-common/src/types.ts

```diff
@@ -132,3 +134,66 @@ export enum BalanceType {
   Solana = 'solana',
   Cosmos = 'cosmos',
 }
+
+export enum SolidityType {
+  address = "address",
+  uint256 = "uint256",
+  uint128 = "uint128",
+  uint64 = "uint64",
+  uint32 = "uint32",
+  uint8 = "uint8",
+  uint = "uint",
+  bytes32 = "bytes32",
+  boolean = "bool",
+  string = "string",
+  bytes = "bytes",
+  tuple = "tuple",
+}
+
+export enum SolidityInternalType {
+  address_payable = "address payable",
+  contract = "contract",
+  enum = "enum",
+  struct = "struct",
+  address = "address",
+  uint256 = "uint256",
+  uint128 = "uint128",
+  uint64 = "uint64",
+  uint32 = "uint32",
+  uint8 = "uint8",
+  uint = "uint",
+  bytes32 = "bytes32",
+  boolean = "bool",
+  string = "string",
+  bytes = "bytes",
+}
+
+export enum SolidityStateMutability {
+  pure = "pure",
+  view = "view",
+  nonpayable = "nonpayable",
+  payable = "payable",
+}
+
+export type Address = string;
```

Comment:
This might have namespace collisions elsewhere? why is this necessary?

Reply:
if we're not using these, we should probably remove them for the moment

Reply:
removing